### PR TITLE
Release guide

### PR DESCRIPTION
**Pull request by lukaszlenart**
Documents a very complicated release process ;-)

### CODE REVIEW

**Comment by lukaszlenart** [comment]
@pdyraga can you review this release process? Please ignore the two first commits, they are in `master` right now, I just forgot to push before branching off of the `master` :/

**Comment by arigoldx** [comment]
@pdyraga @lukaszlenart fyi I was in here trying to figure out why "Service Master" wasn't matching "Service Master to Univar via Coupa HTTPS cXML" and noticed this open PR.

File: README.md [hunk 1]
```diff
@@ -173,3 +173,20 @@ $ npm run-script dist
...
 all the time, that means you should rarely, if ever, need to invoke dist directly, right?
+
+## Release process
+
+To release a new version of `elemica-suggest` follow the below steps:
+
+1. fetch and checkout the `master` branch
+2. remove `-SNAPSHOT` suffix inside `elemica-suggest.coffee`
+3. prepare a dist running `npm run-script dist`
+4. commit both changed files, i.e. `elemica-suggesr.coffee` & `dist/elemica-suggest.js` with a message `Prepares release X.X.X`
```
Comment (pdyraga) [inline README.md]: `elemica-suggest.coffee`

Comment (lukaszlenart) [reply README.md]: 👍

**Comment by pdyraga** [comment]
👍

**Pull request merged**